web/client/api/__tests__/WMS-test.js

```diff
@@ -175,6 +174,18 @@ describe('Test correctness of the WMS APIs', () => {
             }
         });
     });
+
+    it('GetRecords transform SRS List to uppercase', (done) => {
+        API.getRecords('base/web/client/test-resources/wms/GetCapabilities-1.3.1.xml', 0, 2, '').then((result) => {
+            try {
+                expect(result.records[0].SRS).toNotEqual(['EPSG:3857', 'EPSG:4326', 'CRS:84']);
```

Comment:
Probably you mean to use `toEqual`?

Reply:
Sorry forgot this when I was testing to make sure the test fails